src/pages/Checklist/index.tsx

```diff
@@ -173,43 +204,46 @@ export const Checklist = () => {
               <Text className="checkbox-label">
                 I have forwarded the necessary ports from my router to the
                 correct machine(s). Only open the ports that apply to your
-                installation (default ports listed):{' '}
+                installation.
               </Text>
             }
           />
-          <ul className="sub-checklist-item">
-            <li>
-              <Text>
-                <strong>geth</strong>: Port 30303 TCP/UDP
-              </Text>
-            </li>
-            <li>
-              <Text>
-                <strong>Lighthouse</strong>: Port 9000 TCP/UDP
-              </Text>
-            </li>
-            <li>
-              <Text>
-                <strong>Nimbus</strong>: Port 19000 TCP/UDP
-              </Text>
-            </li>
-            <li>
-              <Text>
-                <strong>Prysm</strong>: Port 12000 UDP, Port 13000 TCP
-              </Text>
-            </li>
-            <li>
-              <Text>
-                <strong>Teku</strong>: Port 9001 TCP/UDP
-              </Text>
-            </li>
-          </ul>
+          <PortTable>
+            <thead>
+              <tr>
+                <th>Validator</th>
```

Comment:
`Validator` -> `Service` since geth is not eth2 validator. 

```suggestion
                <th>Service</th>
```